dockerfile/README.md

```diff
@@ -1,33 +1,49 @@
 # Dockerfile for WSO2 Identity Server #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+The Dockerfile defines the resources and instructions to build the Docker image for WSO2 Identity Server 5.3.0.
 
-## Try it out
-Quick steps to build the WSO2 Identity Server docker image and run in your local machine
+## Build and run
+Steps to build the WSO2 Identity Server 5.3.0 docker image and run in your local machine.
 
-The cloned local copy of WSO2 Dockerfiles will be referred as `DOCKERFILES_HOME`.
+The local copy of the `dockerfile` directory will be referred as `DOCKERFILE_HOME`.
 
-* Add product packs and dependencies
-    - Download and copy JDK 1.7 ([jdk-7u80-linux-x64.tar.gz](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)) pack to `<DOCKERFILES_HOME>/common/provision/default/files`.
-    - Download the WSO2 Identity Server zip file (http://wso2.com/products/identity-server/) and copy it to `<DOCKERFILES_HOME>/common/provision/default/files`.
+* Create a directory named `files` inside `DOCKERFILE_HOME`
+    - This will result in a directory structure `DOCKERFILE_HOME/files`.
+
+* Add the JDK and WSO2 Identity Server distributions
+    - Download JDK 1.8 (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and copy it to `<DOCKERFILE_HOME>/files`.
+    - Download the WSO2 Identity Server 5.3.0 distribution (https://wso2.com/identity-and-access-management) and copy it to `<DOCKERFILE_HOME>/files`.
```

Comment:
Would it be better to mention to download the IS distribution via WUM? Maybe those steps can be listed here, WDYT?

Reply:
+1, provided a link to the WUM docs.